### PR TITLE
Japanese translation

### PR DESCRIPTION
**Pull request by shu-mutou**
Add Japanese translation file `i18n/messages.ja.xlf`.

### CODE REVIEW

**Comment by shu-mutou** [comment]
/hold
until #3971 merged.

File: aio/develop/Dockerfile [hunk 1]
```diff
@@ -35,7 +35,7 @@ RUN curl -sL https://deb.nodesource.com/setup_11.x | bash - \
 	&& apt-get clean
 
 # Install dependencies. This will take a while.
-RUN npm install -g npm@latest gulp
+RUN npm install -g npm@latest gulp @angular/cli@latest
```
Comment (maciaszczykm) [inline aio/develop/Dockerfile]: `latest` can be problematic with the new releases. Let's keep that in mind for the future.

Comment (shu-mutou) [reply aio/develop/Dockerfile]: Thanks! Versions for gulp and angular/cli were specified same as written in package.json in #3971

**Comment by maciaszczykm** [comment]
Thank you for this huge effort, looks great.

**Pull request merged**